src/client/pages/ConsentsConfirmation.tsx

```diff
@@ -114,13 +118,13 @@ export const ConsentsConfirmation = ({
   success,
   returnUrl,
   productConsents,
-  optedOutOfProfiling,
+  optedIntoProfiling,
   subscribedNewsletters,
   geolocation,
 }: ConsentsConfirmationProps) => {
   const autoRow = getAutoRow(1, gridItemColumnConsents);
   const anyConsents =
-    !optedOutOfProfiling ||
+    !optedIntoProfiling ||
```

Comment:
I think this should just be `optedIntoProfiling`. In storybook this is leading to the "Please find below a summary of your settings" message showing, even when there is nothing to show.

Reply:
thanks. Sorted.